### PR TITLE
Add E2E tests for container-manager-service

### PR DESCRIPTION
**Pull request by mbarnes**
This adds a new `TrivialContainerManager` class to aid testing; it just tracks node addresses in memory.

Even with this, however, container manager E2E coverage isn't great.   The REST API offers very limited visibility.

Would be good to add a `kombu.connection.Connection` instance to the test environment so we can query services directly when necessary to help improve coverage, but that's a separate task for later.

### CODE REVIEW

**Comment by mbarnes** [comment]
@ashcrow I think this is ready for review.

Note it requires https://github.com/projectatomic/commissaire-http/pull/72 first.

**Comment by ashcrow** [comment]
bot, retest this please

File: features/steps/containermgr.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+@then('the host status will include container manager details')
+def impl(context):
+    container_manager_status = context.request.json().get('container_manager')
+    assert type(container_manager_status) is dict
+    assert len(container_manager_status) > 0
+
+@then('the host status will not include container manager details')
+def impl(context):
+    container_manager_status = context.request.json().get('container_manager')
+    assert type(container_manager_status) is dict
```
Comment (ashcrow) [inline features/steps/containermgr.py:45]: nit: No change needed. Could be ```assert isinstance(container_manager_status, dict)```

File: src/commissaire/containermgr/trivial.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2017  Red Hat, Inc
```
Comment (ashcrow) [inline src/commissaire/containermgr/trivial.py:1]: Since this is only ever to be used in testing could we move this into the ``test/`` directory?

**Review by ashcrow — CHANGES_REQUESTED** [review]
One request on moving an item into ``test``. Otherwise, looks good!

File: src/commissaire/containermgr/trivial.py (hunk 2 above)
Comment (mbarnes) [inline src/commissaire/containermgr/trivial.py]: I'll have to generate a custom container-manager config file for the non-standard module path, but I think so.  (We generate one already for the storage service.)

Comment (ashcrow) [reply src/commissaire/containermgr/trivial.py:1]: If that isn't much extra work let's do that. If it is a pain then let me know and I won't block.

Comment (mbarnes) [reply src/commissaire/containermgr/trivial.py:1]: @ashcrow: How strongly do you feel about that?

Because it's imported as a plugin, it would have to be specified as a full module path in a `ContainerManagerConfig`.  And in order for that to work, I'd have to hack the `PYTHONPATH` in the `container-manager-service` environment to point to the `/vagrant/commissaire/features` directory (where `trivial.py` would live), and that then involves hacking its `.service` file in the `Vagrantfile`.

In other words it's getting pretty ugly, unless I'm missing a simpler way.

**Review by ashcrow — APPROVED** [review]
